### PR TITLE
Fix warning/note for preview/stable

### PR DESCRIPTION
**Pull request by premkumr**
- On stable when preview is behind stable - do nothing.
- For preview: warning box and link to the stable
- For stable: note with link to preview
- For older LTS/STS other than stable :  note with link to preview and to the latest stable

### CODE REVIEW

File: docs/layouts/partials/earlier-version-warning.html [hunk 1]
```diff
@@ -1,27 +1,72 @@
...
+                  For the latest stable version, go to <a href="{{ $stableUrl }}">stable ({{ $stableVersion }})</a> version.
+                  {{- end -}}
+                </p>
+              </div>
+            {{- end -}}
+          {{- else if eq .alias "preview" -}}
+            {{/** If a stable page does not exist, this is an unversioned page!!!! **/}}
+            {{- if $stableExists -}}
+              <div class="admonition warning">
+                <p>This page documents the preview ({{ $previewVersion }}) version that is to be used for testing and development with the
```
Comment (ddhodge) [inline docs/layouts/partials/earlier-version-warning.html:55]:
```suggestion
                <p>This page documents the preview ({{ $previewVersion }}) version. Preview includes features under active development and is for development and testing only.
```

File: docs/layouts/partials/earlier-version-warning.html [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+                  latest features only. For production, use the <a href="{{ $stableUrl }}"> stable ({{ $stableVersion }})</a> version.</p>
```
Comment (ddhodge) [inline docs/layouts/partials/earlier-version-warning.html:56]:
```suggestion
                 For production, use the <a href="{{ $stableUrl }}"> stable ({{ $stableVersion }})</a> version.</p>
```

File: docs/layouts/partials/earlier-version-warning.html [hunk 3]
```diff
@@ -1,27 +1,72 @@
...
-            <!-- <p class="admonition-title">Attention</p> -->
-            <p>This page documents a stable (production) version. For testing and development with the latest features, use the <a href="{{ $previewUrl }}">preview {{ $previewVersion }} version.</a></p>
+            {{/** Do nothing if preview does not exist for this file **/}}
+            {{- if $previewExists -}}
+              <div class="admonition note">
+                <p>This page documents a stable (production) version. For testing and development with the latest features,
+                  use the <a href="{{ $previewUrl }}">preview ({{ $previewVersion }}) version.</a>
+                  {{/** For older stable versions, Add link to latest stable **/}}
+                  {{ if not (eq .alias "stable") }}
+                  For the latest stable version, go to <a href="{{ $stableUrl }}">stable ({{ $stableVersion }})</a> version.
```
Comment (ddhodge) [inline docs/layouts/partials/earlier-version-warning.html:46]:
```suggestion
                  For the latest stable version, see <a href="{{ $stableUrl }}">({{ $stableVersion }})</a>.
```

**Comment by premkumr** [comment]
Closing this PR as warn statements are not properly printed.

**Pull request closed without merging**